evennia/contrib/rpg/rpsystem/rpsystem.py

```diff
@@ -202,7 +202,7 @@ class Room(ContribRPRoom):
 _RE_REF = re.compile(r"\{+\#([0-9]+[\^\~tv]{0,1})\}+")
 
 # This regex is used to quickly reference one self in an emote.
-_RE_SELF_REF = re.compile(r"/me|@", _RE_FLAGS)
+_RE_SELF_REF = re.compile(r"/me\b|@", _RE_FLAGS)
```

Comment:
I think the problem I did not use `\b` here is because in English you want to self-reference with possessive form, so you want to be able to write something like `/me's`: 

    There is a frown as the other person tries to touch /me's sword. "Don't do that, he hisses."

In the old style of using the algorithm, this worked out due to how it was split out into components, here one probably needs to handle this specifically.